geom.js

```diff
@@ -1,26 +1,78 @@
 class Rectangle {
-  constructor(length, width) {
-    this.length = length;
-    this.width = width;
+  constructor (length, width) {
+    this.length = length
+    this.width = width
+  }
+  isSquare () {
+    if (this.length === this.width) {
+      return true
+    }
+  }
+  area () {
+    return this.length * this.width
+  }
+  perimeter () {
+    return (this.length * 2) + (this.width * 2)
   }
 }
 
-
 class Triangle {
-  constructor(sideA, sideB, sideC){
-    this.sideA = sideA;
-    this.sideB = sideB;
-    this.sideC = sideC;
+  constructor (sideA, sideB, sideC) {
+    this.sideA = sideA
+    this.sideB = sideB
+    this.sideC = sideC
+  }
+  isIsosceles () {
+    if (this.sideA === this.sideB || this.sideB === this.sideC || this.sideC === this.sideA) {
+      return true
+    } else {
+      return false
+    }
+  }
+  isEquilateral () {
+    if (this.sideA === this.sideB && this.sideA === this.sideC) {
+      return true
+    } else {
+      return false
+    }
+  }
```

Comment:
The pattern `if (condition) {return true} else {return false}` can always be simplified by just returning the value of the conditional check:
```js
isEquilateral () {
  return (this.sideA === this.sideB && this.sideA === this.sideC) 
}


```